tracer.go

```diff
@@ -1,48 +1,72 @@
 package zipkin
 
 import (
+	"context"
+	"sync/atomic"
 	"time"
 
-	"github.com/openzipkin/zipkin-go/kind"
+	"github.com/openzipkin/zipkin-go/idgenerator"
+	"github.com/openzipkin/zipkin-go/model"
+	"github.com/openzipkin/zipkin-go/propagation"
+	"github.com/openzipkin/zipkin-go/transport"
 )
 
 // Tracer is our Zipkin tracer implementation.
 type Tracer struct {
-	options TracerOptions
+	noop                 int32
```

Comment:
I am curious why this is not Boolean.

Reply:
it needs to be goroutine safe so it uses atomic bool logic where 1 = true and 0 = false (Go does not have a native atomic bool so int32 is used)

Reply:
adding comment about atomic bool usage